User Story 3: Task 1: Implement Contact Form Error Page Structure

HenrikLeutscher opened the pull request:
Created a Seperate HTML file for displaying error handling on the contact form.

Ensured a structure that follows a logical layout for error handling

Implemented / replaced the submit button with a Try Again button once the form error initiates. This button navigates back to the original contact.html

Replaced header with student header.

HenrikLeutscher committed:
User Story 3: Task 1: Implement Contact Form Error Page Structure

Created a Seperate HTML file for displaying error handling on the contact form.

Ensured a structure that follows a logical layout for error handling

Implemented / replaced the submit button with a Try Again button once the form error initiates. This button navigates back to the original contact.html

Replaced header with student header.

kevindjenhansen reviewed (changes requested):
Great work! I've just noticed the Try Again button doesn't work when on mobile?

HenrikLeutscher committed:
Mobile Accessibility

Made the form accessible on mobile view, without anything overlapping it making it disfunction.

kevindjenhansen reviewed (approved):
Looks good, great work!
